### DISCUSSION TITLE
Docu like ReadTheDocs / Offering a objects.inv

### DISCUSSION THREAD

**Discussion by buhtz** [borntyping/python-colorlog#133] [Support] [answered]
More a question then a bug.
Am I right that there is no ReadTheDocs project or something similar for "python-colorlog" available?
So there is no "ojbects.inv" file available I could use as an intersphinx reference (e.g. in PyDoctor or Sphinx)?

**Comment by borntyping** [comment, marked as answer]
I'm not familiar with intersphinx, but you're correct that there's no documentation in ReadTheDocs or Sphinx formats. The entirely of the documentation is the `README.md` file and the examples in `docs/`.
